app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -388,7 +388,7 @@ def import_uploads
     param :id, :identifier, :required => true
     def gpg_key_content
       if @repository.gpg_key && @repository.gpg_key.content.present?
-        render(:text => @repository.gpg_key.content, :layout => false)
+        render(:plain => "#{@repository.gpg_key.content}", :layout => false)
```

Comment:
the "#{}"  shouldn't actually be needed, `@repository.gpg_key_content` should work just fine